File: codelist/python/07file/readWriteFile.py
#/user/bin/env python   

# -*- coding:utf-8 -*-

"this is a test module for file"
 
__author__ = 'mxl'

readf = open("input.txt","r")
lines = readf.readlines()
for line in [i.strip() for i in lines]:
    print line
readf.close()

with open("output.txt","w") as wf:
    wf.writelines(lines)

File: codelist/Perl/readwritefile.pl
open FOUT,'>','C:\Users\xianlong.mxl\Desktop\output.txt';
open (MYFILE, 'C:\Users\xianlong.mxl\Desktop\input.txt') || die ("Could not open file"); 
while (<MYFILE>) {#利用管道从文件读入
	chomp;#除去行尾换行符 
	print FOUT $_;
	print FOUT "\n";
}
print "done！";

close MYFILE;
close FOUT;